packages/core/admin/admin/src/content-manager/pages/ListView/components/ViewSettingsMenu.tsx

```diff
@@ -65,7 +67,11 @@ const ViewSettingsMenu = ({ slug }: ViewSettingsMenuProps) => {
                 variant="secondary"
                 as={NavLink}
                 // @ts-expect-error – inference from the as prop does not work in the DS.
-                to={'configurations/list'}
+                to={`configurations/list${
+                  query.plugins
+                    ? `?${stringify({ plugins: query.plugins }, { encode: false })}`
+                    : ''
+                }`}
```

Comment:
I'd start using `to` as it's object form – https://reactrouter.com/en/main/components/link see `Type declaration` on the page.